documentation/docs/30-advanced/70-packaging.md

```diff
@@ -132,7 +132,7 @@ You should ship type definitions for your library even if you don't use TypeScri
 
 If you have something else than a root export however — for example providing a `your-library/foo` import — you need to take additional care for providing type definitions. Unfortunately, TypeScript by default will _not_ resolve the `types` condition for an export like `{ "./foo": { "types": "./dist/foo.d.ts", ... }}`. Instead, it will search for a `foo.d.ts` relative to the root of your library (i.e. `your-library/foo.d.ts` instead of `your-library/dist/foo.d.ts`). To fix this, you have two options:
 
-The first option is to require people using your library to set the `moduleResolution` option in their `tsconfig/jsconfig.json` to `bundler` (available since TypeScript 5, the best and recommended option in the future), `node16` or `nodenext`. This opts TypeScript into actually looking at the exports map and resolving the types correctly.
+The first option is to require people using your library to set the `moduleResolution` option in their `(tsconfig|jsconfig).json` to `bundler` (available since TypeScript 5, the best and recommended option in the future), `node16` or `nodenext`. This opts TypeScript into actually looking at the exports map and resolving the types correctly.
```

Comment:
```suggestion
The first option is to require people using your library to set the `moduleResolution` option in their `tsconfig.json` (or `jsconfig.json`) to `bundler` (available since TypeScript 5, the best and recommended option in the future), `node16` or `nodenext`. This opts TypeScript into actually looking at the exports map and resolving the types correctly.
```

Reply:
yes, I agree with @dummdidumm's suggestion

If you leave "allow edits by maintainers" checked, which it is by default but appears to have been turned off, it would allow us to make that change and merge this PR without you needing to do anything. Since it's been disabled we'll need you to make the change

Reply:
@benmccann I made the change. 

Sorry, where is "allow edits by maintainers"? Is this a setting on my fork?

Reply:
Never mind, its a setting on the organization. In future I'll fork to my personal account.

Reply:
It's on the bottom of the righthand sidebar

![Screenshot from 2023-04-13 09-35-46](https://user-images.githubusercontent.com/322311/231826628-518649b8-e10d-4fbf-bb38-68b154d4b2ae.png)

There's also a checkbox when you create the PR. It might save the setting from the last time you changed it, so if you turn it off once, it might default to off from then on until you turn it back on

Reply:
it's probably not there anymore because the PR has already been merged